easybuild/easyblocks/b/berkeleygw.py

```diff
@@ -135,7 +147,8 @@ def build_step(self):
             self.cfg.update('buildopts', 'MKLPATH="%s"' % mkl)
 
         fftw = get_software_root('FFTW')
-        if mkl or fftw:
+        fujitsufftw = get_software_root('FFTW3-sve')
+        if mkl or fftw or fujitsufftw:
```

Comment:
Hmm, shouldn't we check this differently? Via `self.toolchain.FFTW_API_VERSION`?

I would leave the `if mkl or fftw` in place though, so do this?

```
if mkl or fftw or self.toolchain.FFTW_API_VERSION == '3':
```

The FFTW variant in https://github.com/easybuilders/easybuild-easyconfigs/pull/12863/ should be discriminated via a `versionsuffix` anyway, I think...